backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc

```diff
@@ -26,6 +26,29 @@ include::modules/oadp-backing-up-pvs-csi.adoc[leveloffset=+1]
 include::modules/oadp-backing-up-applications-restic.adoc[leveloffset=+1]
 include::modules/oadp-using-data-mover-for-csi-snapshots.adoc[leveloffset=+1]
 
+[id="oadp-cleaning-up-after-data-mover-1-1-backup"]
+== Cleaning up after a backup using Data Mover with OADP 1.1._x_
```

Comment:
What is the "x" here supposed to be? Is it a replaceable value (if so, then it should be the <x> format). Or if it's really not necessary, like it's any z-stream version, could you just leave it off and say 1.1?